src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -100,8 +99,9 @@ public void getPersonList_modifyList_throwsUnsupportedOperationException() {
             return persons;
         }
 
+        // TODO:
```

Comment:
Is this ```//TODO``` comment still needed?

Reply:
Yes, I have not added in test cases yet, will try to finish it by 1.2b

Reply:
Alright!